documentation/cli/docs/02_wallet.md

```diff
@@ -81,32 +84,35 @@ When just initialised, the wallet has no account yet, use the `new` command to c
 
 #### Options
 
-| Name                 | Description                                    | Default                             | Environment Variable | Example                 |
-| -------------------- | ---------------------------------------------- |------------------------------------ |----------------------| ----------------------- |
-| `--mnemonic` (`-m`)  | Initialises a wallet from an existing mnemonic | ✘                                   | ✘                    |"aunt middle impose ..." |
-| `--node-url` (`-n`)  | Sets the node to connect to                    | https://api.testnet.shimmer.network | `NODE_URL`           | http://localhost:14265  |
-| `--coin-type` (`-c`) | Sets the coin type associated with the wallet  | 4219 (=Shimmer)                     | ✘                    | 4218(=IOTA)             |
+| Name                           | Description                                      | Default                               | Environment Variable | Example                   |
+| ------------------------------ | ------------------------------------------------ |-------------------------------------- |--------------------- | ------------------------- |
+| `--mnemonic-file-path` (`-m`)  | Sets the path to a file containing mnemonics     | ✘                                     | ✘                    | my_mnemonic.txt           |
+| `--node-url` (`-n`)            | Sets the node to connect to                      | <https://api.testnet.shimmer.network> | `NODE_URL`           | <http://localhost:14265>  |
```

Comment:
Why the `<>`?

Reply:
this is the auto-linter, Should I remove it?

Reply:
I would remove it, otherwise one might think one has to also write <> in the cli command which will then fail

Reply:
I would remove, yes

Reply:
ok, removed